includes/class-cron.php

```
@@ -393,4 +402,77 @@ public function database_cleanup_cron() {
+		if ( empty( $url ) ) {
+			return;
+		}
+
+		if ( ! class_exists( 'PerformanceOptimise\Inc\Telemetry' ) ) {
+			require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry.php';
+		}
+		if ( ! class_exists( 'PerformanceOptimise\Inc\Telemetry_Table' ) ) {
+			require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry-table.php';
+		}
+
+		$result = Telemetry::scan( $url, 'scheduled' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Telemetry' in method 'run_scheduled_telemetry'. (undefined)

(StaticAccess)